### ISSUE TITLE
App Layout: Bottom sheet padding missing (and landscape mode issue)

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-android#7027]
### Steps to reproduce

When I open the bottom sheet there's no padding above all chats, it looks unpolished 

Also, when in landscape mode the bottom sheet only shows all chats

| In product | In figma |
|---|---|
| ![ima_915cfc3](https://user-images.githubusercontent.com/89144281/188593986-203cd7ae-fa9c-46b4-b7f0-ebf9714341a2.jpeg) | <img width="1500" alt="Screenshot 2022-09-06 at 10 03 35" src="https://user-images.githubusercontent.com/89144281/188594278-0b4de16d-6755-45e5-9d04-73a8d350749a.png"> |

![image](https://user-images.githubusercontent.com/89144281/188595148-d590cdcb-4358-43b0-9ad3-e92cfebf6ab3.jpeg)

**Comment by ericdecanini** [comment]
Looking back at the designs, it looks like what we want to do is add back the "CHANGE SPACE" header. I'm not sure where but I recall a decision was made a little while ago to remove this.

@amshakal can you confirm this please

**Comment by amshakal** [comment]
Yeah, we should add a simple header (non clickable) saying 'Change Space'. it'll make this sheet look polished
